src/sentry/integrations/base.py

```diff
@@ -23,8 +25,16 @@
 from .constants import ERR_UNAUTHORIZED, ERR_INTERNAL, ERR_UNSUPPORTED_RESPONSE_TYPE
 from sentry.models import Identity, OrganizationIntegration
 
+
+FeatureDescription = namedtuple('FeatureDescription', [
+    'description',  # A markdown description of the feature
+    'featureGate',  # A IntegrationFeature that gates this feature
```

Comment:
python usually uses snake_case. Is this to make serialization easier?

Reply:
`gate` would be a shorter name. `featureGate` duplicate's the noun's name. :bike::house:

Reply:
> python usually uses snake_case. Is this to make serialization easier?

Yeah, the others metadata namedtuple keys should be updated as well.

> `gate` would be a shorter name. `featureGate` duplicate's the noun's name. 🚲🏠

Yeah.. it's a little confusing, we actually conflate `feature` here quite a bit. Since this is a description of a feature, versus the `IntegrationFeature` which are the actual keys representing general integration features, which then map over to organization "features", which come from "plan features". 🤷‍♂️